webview/src/experiments/components/table/styles.module.scss

```diff
@@ -369,7 +375,7 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
       background-color: $header-bg-color;
     }
 
-    &:not(.rowSelected) > *:first-child:after {
```

Comment:
Notice that this line shouldn't need a `:not(.rowSelected)` since it's inside a `headRow`, which can't get selected :)